package.json

```
@@ -6,11 +6,13 @@
   "browser": "./lib/index",
   "module": "./index",
   "react-native": "./index",
+  "types": "./index.d.ts",
   "scripts": {
     "build": "babel -d lib index.js",
     "prepublishOnly": "npm run build",
     "lint": "eslint --fix *.js",
-    "test": "nyc mocha --require setup-env test/*.test.js",
+    "typings:test": "tsc --noEmit ./index.d.ts",
+    "test": "nyc mocha --require setup-env test/*.test.js && npm run typings:test",
```

Let's split out the mocha tests if we're doing this
```suggestion
    "test:mocha": "nyc mocha --require setup-env test/*.test.js",
    "test": "npm run test:mocha && npm run test:types",
```

---

Sounds good 👍